build.gradle

```diff
@@ -41,23 +41,37 @@ if (System.env.CIRCLE_TEST_REPORTS) {
 }
 
 group 'com.palantir.gradle.docker'
-version gitVersion()
+version '0.6.0.1'
 
 task sourceJar(type: Jar) {
     from sourceSets.main.allSource
     classifier 'sources'
 }
-  
+
 publishing {
     publications {
-        bintray(MavenPublication) {
+//        bintray(MavenPublication) {
+//            from components.java
+//            artifact(sourceJar)
+//            artifact(publishPluginJavaDocsJar)
+//        }
+        maven(MavenPublication) {
             from components.java
             artifact(sourceJar)
             artifact(publishPluginJavaDocsJar)
         }
     }
+    repositories {
+        maven {
+            credentials {
+                username 'deployment'
+                password 'MXJmqxvTGve9s6q9y7Ks'
```

Comment:
pretty sure you don't want to commit the set of changes to this file, and you might want to think about changing this exposed password.